spec/lib/response_comparator_spec.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require "spec_helper"
+require "faraday"
```

Comment:
Is Faraday needed here?

Reply:
Unfortunately it is, so that we can create the instance_double of a Faraday::Response on line 11. It's not a problem when you run the whole test suite as it's loaded by other files, but when you are just running this one file, it's needed.